ocrd_anybaseocr/ocrd-tool.json

```
@@ -80,10 +80,8 @@
       "parameters": {
         "force":        {"type":"boolean",                      "default":true, "description": "overwrite existing files"},
         "imgresize":    { "type": "string",                     "default": "resize_and_crop", "description": "run on original size image"},
-        "pix2pixHD":    { "type": "string", "default":"", "description": "Path to pix2pixHD library"},
-        "model_name":	{ "type": "string", "default":"models", "description": "name of dir with trained pix2pixHD model (latest_net_G.pth)"},
-        "checkpoint_dir":   { "type": "string", "default":"./", "description": "Path to where to look for dir with model name"},
-        "gpu_id":       { "type": "number", "format": "integer", "default": 0,    "description": "gpu id"},
+        "model_path":	{ "type": "string", "description": "Path to the trained pix2pixHD model (Download from https://cloud.dfki.de/owncloud/index.php/s/3zKza5sRfQB3ygy/download)", "required": true, "cacheable": true, "content-type": "application/vnd.pytorch"},
+        "gpu_id":       { "type": "number", "format": "integer", "default": -1,   "description": "device ID of CUDA GPU to use. Set -1 to use CPU only."},
         "resizeHeight": { "type": "number", "format": "integer", "default": 1024, "description": "resized image height"},
         "resizeWidth":  { "type": "number", "format": "integer", "default": 1024, "description": "resized image width"},
         "operation_level": {"type": "string", "enum": ["page","region", "line"], "default": "page","description": "PAGE XML hierarchy level to operate on"}
```

The implementation never really supported _line_ level, and there are no models trained for _region_ level specifically...

```suggestion
        "operation_level": {"type": "string", "enum": ["page","region"], "default": "page","description": "PAGE XML hierarchy level to operate on (should match what model was trained on!)"}
```